corehq/apps/reminders/models.py

```diff
@@ -1557,9 +1562,38 @@ def _recipient_lookup(self):
             return Group.get(handler.user_group_id)
         elif handler.recipient == RECIPIENT_LOCATION:
             return handler.locations
+        elif handler.recipient == RECIPIENT_CASE_OWNER_LOCATION_PARENT:
+            """
+            This is pretty specific right now which is why it's behind a one-off
+            feature flag. When I move reminders to postgres I'm planning on
+            expanding the model to make this kind of recipient choice configurable.
+            """
+
+            # Get the case
+            case = self.case
+            if not case:
+                return None
+
+            # Get the case owner, which we always expect to be a mobile worker in
+            # this one-off feature
+            owner = self.case_owner
+            if not isinstance(owner, CommCareUser):
+                return None
+
+            # Get the case owner's location
+            owner_location = owner.sql_location
+            if not owner_location:
+                return None
+
+            # Get that location's parent location
+            parent_location = owner_location.parent
```

Comment:
true, could have probably consolidated the last couple lookups that way. it felt better for the owner lookup to explictly check that it's a CommCareUser since it can also be a WebUser, Group, or Location